Back drop fix.

umangdarji-dev opened the pull request.

ankitcf commented:
Why are we making this change ? Modal should close on clicking the backdrop

On Mon, Feb 17, 2020 at 5:02 PM Kunal Khandelwal <notifications@github.com>
wrote:

> *@kunal15595* approved this pull request.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/curefit/react-crux/pull/96?email_source=notifications&email_token=AFC7FWB6VQRXXNGFICDPUJDRDJYTDA5CNFSM4KWPVYPKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCVYB6LA#pullrequestreview-359669548>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFC7FWHOL3CRFTHD4TK7YZTRDJYTDANCNFSM4KWPVYPA>
> .
>

kunal15595 commented:
Even I use this behaviour, but I have seen it becoming a problem for a lot
of new users.
We can always close the modal by closing on cross button.

On Mon, Feb 17, 2020 at 5:05 PM Ankit Gupta <notifications@github.com>
wrote:

> *@ankitcf* requested changes on this pull request.
>
> We should close the modal on clicking the backdrop
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/curefit/react-crux/pull/96?email_source=notifications&email_token=ABKQAKTRO7GPOCEJGU3GY2TRDJY6TA5CNFSM4KWPVYPKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCVYCKTQ#pullrequestreview-359671118>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABKQAKTI2DVZTWUOGZAAGQ3RDJY6TANCNFSM4KWPVYPA>
> .
>


-- 
Kunal Khandelwal
Software Engineer - API

umangdarji-dev commented:
> We should close the modal on clicking the backdrop

We have added all the information in modal and then if we accidentally clicked on backdrop it will close the modal and our data what we have filled previously will lost that's why I have added this.

ankitcf commented:
This is the default behaviour for all bootstrap modal in the world. I am
assuming they have done some research and have deeper insights than the 3
of us.

If a particular dashboard wants it, they should be able to pass this as an
option.

Role of react crux is not to fix bootstraps mistakes

On Mon, 17 Feb, 2020, 5:09 pm Umang Darji, <notifications@github.com> wrote:

> We should close the modal on clicking the backdrop
>
> We have added all the information in modal and then if we accidentally
> clicked on backdrop it will close the modal and our data what we have
> filled previously will lost that's why I have added this.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/curefit/react-crux/pull/96?email_source=notifications&email_token=AFC7FWB5QTLJ47NADBJN7ITRDJZQ7A5CNFSM4KWPVYPKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEL6DIYY#issuecomment-586953827>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFC7FWE3ROG4VDSDPQOPT3DRDJZQ7ANCNFSM4KWPVYPA>
> .
>
